src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
+
+    if follower.detect() == False:
+        print('Detect False')
+        follower._speed = follower.LEADER_MAX_SPEED
+        turn = turn + 35
+    else:
+        print("Detect True")
```

Comment:
Again, we shouldn't be printing and there shouldn't be any dead code commented out.